packages/grafana-ui/src/types/datasource.ts

```diff
@@ -542,3 +542,13 @@ export interface DataSourceSelectItem {
   meta: DataSourcePluginMeta;
   sort: string;
 }
+
+export interface AnnotationQueryRequest<MoreOptions = {}> {
```

Comment:
Did not find this in the code base so created one based on the datasources.md doc